js/views/settingsVw.js

```diff
@@ -91,6 +114,15 @@ module.exports = Backbone.View.extend({
     this.moderatorCount = 0;
 
     this.fetchModel();
+    
+    $(window).off('keydown.view').bind('keydown.view', function(e) {
+      var char = String.fromCharCode(e.which).toLowerCase(),
+          ctrl = (e.ctrlKey || e.metaKey) && !e.altKey;
+
+      if (ctrl && char == config.keyShortcuts.save) {
+        window.obEventBus.trigger('saveCurrentForm', self._state);
```

Comment:
I think the idea is to keep all the keyboard shortcuts in one place.

Though now that you bring it up, it might make more sense to just put the key binding in each view.